utils/jsonutil/strings.go

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package jsonutil
+
+import "encoding/json"
+
+// ToJson marshals v to a JSON string
+func ToJson(v interface{}) string {
+	bytes, _ := json.Marshal(&v)
```

Comment:
Would be nice to at least log it here and not ignore it. What if I pass a channel here? :)

Reply:
Point taken